src/bidiMapper/scripts/eval.spec.ts

```diff
@@ -0,0 +1,162 @@
+import fs from 'fs/promises';
+import path from 'path';
+import * as chai from 'chai';
+import chaiExclude from 'chai-exclude';
+
+chai.use(chaiExclude);
+
+describe('Evaluator', function () {
+    let EVALUATOR: any;
```

Comment:
I'd prefer to avoid using `any` when the type is actually known. Could you define a small interface type with the evaluate/serialize/deserialize methods?

Reply:
done